.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["bloq", "prettier"],
+  "overrides": [
+    {
+      "extends": ["bloq/typescript", "prettier"],
+      "files": ["website/**/*.{js,ts,tsx}"]
+    },
+    {
+      "extends": ["bloq/typescript", "prettier"],
+      "files": ["*.d.ts"]
```

Comment:
Should we merge these two as the "extends" are the same?

Reply:
_We could_, note that once we get `bloq/vitest` we need to separate them again

Reply:
merged in [aa17bb6](https://github.com/hemilabs/tge-mint-page/pull/1/commits/aa17bb668f86ea420a591dd90e30f608705168f7)